std/digest/package.d

```diff
@@ -1212,3 +1212,112 @@ if (isInputRange!R1 && isInputRange!R2 && !isInfinite!R1 && !isInfinite!R2 &&
         assert(!secureEqual(hex1, hex2));
     }
 }
+
+/**
+ * Used to convert hex string to byte array
+ * Params:
+ *  hex = hex string
+ *  num = count of bytes
+ * Returns: byte array
+ * Example:
+ * ---
+ * ubyte[2] sba = "0xAA".fromHexString!2; // returns static array, no allocating
```

Comment:
I don't think this is a good API - the user has to match the compile-time length to the value.

Reply:
I have any idea how to create a function with nogc and static return  without such parameter

Reply:
> I don't think this is a good API - the user has to match the compile-time length to the value.

api changed